ChildProject/pipelines/samplers.py

```diff
@@ -372,16 +372,32 @@ def _segment_scores(self, recording):
         windows['recording_filename'] = recording['recording_filename']
 
         if self.metric == 'ctc':
-            # NOTE: The original code explicitly chooses for these conversational turn 
-            # types, but we might wish to verify what they are. 
-            segments['is_CT'] = segments['lena_conv_turn_type'].isin(['TIFR', 'TIMR'])
+            if 'lena_conv_turn_type' in segments.columns:
+                # NOTE: This is the equivalent of CTC (tab1) in rlena_extract.R
+                segments['is_CT'] = segments['lena_conv_turn_type'].isin(['TIFR', 'TIMR'])
+            else:
```

Comment:
do the following lines really mimic LENA's cvc ?

Reply:
do you mean cvc or ctc? the code displayed above lines 375-8 is for CTC, right?